Snakefile

```diff
@@ -45,45 +45,29 @@ rule compress:
     shell:
         "pigz < {input} > {output}"
 
-rule:
-    """Cut TES from 5' of R1. TES=AGATGTGTATAAGAGACAG. Discard untrimmed."""
+rule final_trim:
+    """
+    Cut H1691' + TES sequence from 5' of R1. H1691'=CATGACCTCTTGGAACTGTC, TES=AGATGTGTATAAGAGACAG.
+    Cut 3' TES' sequence from R1 and R2. TES'=CTGTCTCTTATACACATCT
+    Discard untrimmed.
+    """
     output:
-        r1_fastq=temp("{dir}/trimmed-b.1.fastq.gz"),
-        r2_fastq=temp("{dir}/trimmed-b.2.fastq.gz")
+        r1_fastq="{dir}/trimmed-c.1.fastq.gz",
+        r2_fastq="{dir}/trimmed-c.2.fastq.gz"
     input:
         r1_fastq="{dir}/unbarcoded.1.fastq.gz",
         r2_fastq="{dir}/unbarcoded.2.fastq.gz"
     log: "{dir}/trimmed-b.log"
     threads: 20
     shell:
         "cutadapt"
-        " -g AGATGTGTATAAGAGACAG"
+        " -a ^CATGACCTCTTGGAACTGTCAGATGTGTATAAGAGACAG...CTGTCTCTTATACACATCT "
```

Comment:
Will the ```--discard-untrimmed``` interpret the 
```
CATGACCTCTTGGAACTGTCAGATGTGTATAAGAGACAG...CTGTCTCTTATACACATCT
```
adapter as two separate adapters? It seems like that from the changes you've shown to the output, but this would be nice to verify.

Reply:
Yeah so `CATGACCTCTTGGAACTGTCAGATGTGTATAAGAGACAG...CTGTCTCTTATACACATCT`is a linked adaptor i.e. has the form `PREFIX...SUFFIX`. In this case it will look for the PREFEX first and cut that out before looking at the SUFFIX. And if no SUFFIX is found the read is still regarded as trimmed. 
Also note that I included a option `--pair-filter 'first'` which means that the filtering will only be applied on the first read, not the second.

Reply:
Yeah, I got that. But nice, I'll proceed to merging.